sklearn/preprocessing/label.py

```diff
@@ -33,18 +34,47 @@
 ]
 
 
-def _encode_numpy(values, uniques=None, encode=False, check_unknown=True):
+def _nan_unique(ar, return_inverse=False, allow_nan=False):
```

Comment:
I think elsewhere in the repo we have an allow_nans parameter. Should probably keep the naming consistent (plural or singular)

Reply:
I do not see any "allow_nans" parameter elsewhere. Did I miss something ?